package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "harvest-strategy-polygon",
```

Comment:
Please update these details for the correct repo

Reply:
Updated package.json file